dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/PullRequests.java

```diff
@@ -41,5 +49,31 @@ public void prepareToCreate(final Namespace ns,
         }
         ResultsProcessor.processResults(skippedRepos, exceptions, log);
     }
+
+    /**
+     * Check if the repository is onboarded to Renovate. The signal we are looking for are
+     * (1) The presence of a file named renovate.json in the root of the repository
+     * (2) Ensuring that the file does not have the key "enabled" set to "false"
+     * @param filePath the name of the file that we are searching for in the repo
+     * @param fork A GitHubContentToProcess object that contains the fork repository that is under process
+     * @return true if the file is found in the path specified and is not disabled, false otherwise
+     */
+    private boolean isRenovateEnabled(String filePath, GitHubContentToProcess fork) throws IOException {
+        try {
+            GHContent fileContent = fork.getParent().getFileContent(filePath);
+            InputStream is = fileContent.read();
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(is));
+            JSONTokener tokener = new JSONTokener(bufferedReader);
+            JSONObject json = new JSONObject(tokener);
+            //If the renovate.json file has the key 'enabled' set to false, it indicates that while the repo has been onboarded to renovate, it has been disabled for some reason
+            if (json.has("enabled") && json.getBoolean("enabled") == false) {
+                return false;
+            }
```

Comment:
I think you can replace this if statement with

```java
return json.optBoolean("enabled", true)
```
If the file exists and enabled is not set it is assumed to be true, otherwise return the value of the enabled field.